src/encoded/static/components/matrix.js

```diff
@@ -170,6 +170,28 @@ const analyzeSubCategoryData = (subCategoryData, columnCategoryType) => {
     return { subCategorySums, maxSubCategoryValue, minSubCategoryValue: minSubCategoryValue - 1 };
 };
 
+/**
+ * Remove spaces from id so it can be accepted as an id by HTML
+ *
+ * @param {string} id
+ * @returns id without space or dash if id is empty
+ */
+const sanitizeId = id => (id ? `${id.replace(/\s/g, '_')}` : '-');
+
+let _navbarHeight = null;
+
+/**
+ * Get height of the navbar
+ *
+ * @returns height of navbar on first call and restores same value until page is garbage collected
+ */
+const getNavbarHeight = () => {
+    if (_navbarHeight === null) {
+        const navbar = document.querySelector('#navbar');
+        _navbarHeight = navbar ? navbar.getBoundingClientRect().height : 0;
+    }
+    return _navbarHeight;
+};
```

Comment:
I’m _personally_ not a fan of using IDs to get the height of React-rendered elements, but I won’t gate this review on that because this is way simpler than the way I would have done it, so I think this method has a valid argument in its favor. Since we have the `<Navigation>` component which can find out the height of the menu using React refs, and can feed that back to `<App>` which can then pass this information in React context (maybe along with other info we don’t yet use, like footer height, navigation color, cart menu visible, etc.). Maybe that’s a mechanism best done in another ticket.